src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -59,12 +59,15 @@
 
         // Estimated gearing constant of 41
         public static final double TURRET_DEGREES_TO_ENCODER_TICKS = 41 * 2048 / 360; // 233
-        public static final double MIN_TURRET_ANGLE = -90;
-        public static final double MAX_TURRET_ANGLE = 90;
+        public static final double MIN_TURRET_ANGLE = -180;
```

Comment:
We should be careful to make sure the turret constants are correct or the do not overwrite values tuned tomorrow.

Reply:
Putting these back since I see Alex changed them to very different values.